lib/core/domain/entities/position.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:math';
+
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'position.freezed.dart';
+
+/// Position of the device on the globe at a moment.
+@freezed
+class Position with _$Position {
+  const Position._();
+
+  const factory Position({
```

Comment:
de manière générale sur les noms de variable si on est en unités SI pas la peine de préciser l'unité (mettre dans la doc de la classe que les grandeurs sont en SI)
uniquement préciser si c'est différent du SI ou si il y a un vrai risque de confusion (pour lat et long en degrés vs radians)

Reply:
Hm je suis assez peu d'accord avec ça, la gestion des unités en développement est un tel enfer que je préfère toujours les mettre, sauf dans le cas où le type le gère, comme `Datetime`, pas besoin de préciser l'unité dans ce cas

Reply:
je suis d'accord que les unités c'est souvent chiant à gérer mais
là on est dans un environnement maitrisé où on peut définir une nomenclature et s'y conformer
si on dit "toutes les variables de grandeur sont en SI sauf mention du contraire" ben toutes les variables sont en SI

après dnas les faits ça me dérange pas de mettre les unités, ce qui m'embête c'est de prendre 15 caractères pour les unités. Genre `speedErrorInMetersPerSecond` je trouve ça inutilement trop long comme nom de variable, `speedErrorMPS` éventuellement je trouve ça plus agréable à lire au final. Parce que quand on devra l'utiliser en restant dans la limite de 80 caractères par ligne ça peut être un peu pénible d'avoir des variables super longues

au minimum je dirais retirer le `In` des variables, lui je le trouve vraiment pas nécessaire. Sur le reste je préférerais avoir le moins de caractères possible mais j'y suis pas non plus suepr attaché, si tu préfères un nom très long parfaitement explicite alors va pour ça